devtools/cmd/migrate-librarian/main.go

```diff
@@ -114,11 +156,9 @@ func deriveLanguage(repoPath string) (string, error) {
 
 func buildConfig(
 	ctx context.Context,
-	librarianState *legacyconfig.LibrarianState,
-	librarianConfig *legacyconfig.LibrarianConfig,
-	lang string) (*config.Config, error) {
+	languageConfig *Config) (*config.Config, error) {
```

Comment:
nit: `languageConfig` got me for a bit too. maybe 'inputConfig' , 'source' or a better name that is clear